src/features/Home/GetStarted/GetStarted.tsx

```diff
@@ -9,6 +9,15 @@ export const GetStarted = () => {
           Get started with our API in 3 simple steps:
         </Text>
         <div className={styles.cardContainer}>
+          <a href='/docs/resources/api-guide/' className={styles.mainPageCard} data-testid='guide'>
```

Comment:
can you please use [Link component](https://docusaurus.io/docs/docusaurus-core#link), thanks :pray:

Reply:
let me know if you got any problem with it.

Reply:
Hello michio, should I change it for all 3 cards or just this one?

Reply:
I changed for all. Let me know if I need to revert the changes back. 😊

Reply:
thanks, yeah we have to update the components for all the cards. :pray: